### PR TITLE
Show "Complete interaction" for draft past meetings

### PR DESCRIPTION
**Pull request by yeahfro**
## Change
This change adds a `Complete interaction` button to draft meetings which are in the past.

## Screenshots

Three main scenarios are covered by the change.

### 1. Future draft meeting - does not have any buttons
Screenshot 2019-05-28 at 10 41 01

### 2. Past draft meeting - has a "Complete interaction" button
Screenshot 2019-05-28 at 10 41 15

### 3. Complete interaction - has an "Edit interaction" button
Screenshot 2019-05-28 at 10 45 31

**Checklist**

- [x] Has the branch been rebased to develop?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [x] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: src/apps/interactions/controllers/details.js [hunk 1]
```diff
@@ -9,13 +9,17 @@ function renderDetailsPage (req, res, next) {
   try {
     const { interaction } = res.locals
     const breadcrumb = capitalize(lowerCase(interaction.kind))
-    const interactionViewRecord = transformInteractionResponseToViewRecord(interaction)
+    const isComplete = interaction.status === 'complete'
```
Comment (rafenden) [inline src/apps/interactions/controllers/details.js]: It's a good practice to avoid magic strings and use constants instead.

Comment (yeahfro) [reply src/apps/interactions/controllers/details.js]: Good spot, I forgot to do this

File: src/apps/interactions/transformers/interaction-response-to-view.js [hunk 2]
```diff
@@ -38,14 +37,13 @@ function transformInteractionResponseToViewRecord ({
   contacts,
   policy_areas,
   policy_issue_types,
-  was_policy_feedback_provided,
   policy_feedback_notes,
```
Comment (jim68000) [inline src/apps/interactions/transformers/interaction-response-to-view.js]: Not clear why we lose `was_policy_feedback_provided` here

Comment (jim68000) [reply src/apps/interactions/transformers/interaction-response-to-view.js]: Turns out it was an unused var

**Pull request merged**